crates/graphql/src/schema/query_root.rs

```diff
@@ -358,8 +366,11 @@ impl QueryRoot {
                 listing_receipts::seller
                     .ne_all(&context.shared.featured_listings_seller_exclusions),
             )
-            .filter(listing_receipts::seller.eq(wallet_totals::address))
-            .order(wallet_totals::followers.desc())
+            .filter(metadata_creators::share.gt(0))
+            .filter(metadata_creators::verified.eq(true))
+            .group_by(listing_receipts::metadata)
+            .group_by(listing_receipts::address)
```

Comment:
`group_by` is a weird undocumented diesel function.  did you mean `distinct_on`?

Reply:
> `group_by` is a weird undocumented diesel function. did you mean `distinct_on`?

no, I am using an aggregator `sum`, which requires `group_by` to work because it allows aggregation, whereas `distinct` would only return a single row for each and bypass the sum

_I think_